spark-3.5-scala-2.13.yaml

```diff
@@ -46,35 +48,38 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 32232e9ed33bb16b93ad58cfde8b82e0f07c0970
 
+  - uses: patch
+    with:
+      patches: make-distribution.patch mvn.patch
+
   - runs: |
       ./dev/change-scala-version.sh 2.13
 
   - uses: maven/pombump
     with:
       properties-file: pombump-properties.yaml
-      pom: pom.xml
 
   - runs: |
-      ./build/mvn -DskipTests -Pscala-2.13 clean package
-
-      mkdir -p ${{targets.contextdir}}/usr/lib/spark
-      mkdir -p ${{targets.contextdir}}/usr/lib/spark/work-dir
-      mv bin/ ${{targets.contextdir}}/usr/lib/spark
-      mv sbin/ ${{targets.contextdir}}/usr/lib/spark
-      mv target ${{targets.contextdir}}/usr/lib/spark
-      cp resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh ${{targets.contextdir}}/usr/lib/spark/
-      cp -R assembly/target/scala-2.13/jars ${{targets.contextdir}}/usr/lib/spark/
-      mv assembly ${{targets.contextdir}}/usr/lib/spark
+      mkdir -p ${{targets.contextdir}}/usr/lib/spark/
+      ./dev/make-distribution.sh --name pyspark-2.13 --pip --tgz -Pscala-2.13 -Phive -Phive-thriftserver -Pyarn
+      mv dist/* ${{targets.contextdir}}/usr/lib/spark/
 
 subpackages:
+  - name: pyspark-2.13
```

Comment:
Nit: This should probably be `pyspark-scala-2.13`

Reply:
Thanks, fixed